pkg/cmd/experimental/diagnostics/options/diagnostics.go

```diff
@@ -1,46 +1,36 @@
 package options
 
 import (
-	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
-	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 	"github.com/spf13/pflag"
-	"io"
-)
 
-// all of the diagnostics commands will bind these options
-type DiagnosticsOptions struct {
-	Diagnostics *util.StringList // named diagnostics to run
-	DiagLevel   int              // show output of this priority or higher
-	DiagFormat  string           // format of output - text/json/yaml
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 
-	Output cmdutil.Output // this is used for discovery and diagnostic output
+	"github.com/openshift/origin/pkg/diagnostics/log"
+)
+
+type RecommendedLoggerOptionsFlags struct {
```

Comment:
typo? I would think they're LoggerOptionsFlags, and one particular instance of that might be "recommended" (or "standard" or whatever).